### ISSUE TITLE
Laravel - Query methods failed on WHERE clause with 2 parameters constraint

### ISSUE THREAD

**Issue by edno** [Codeception/Codeception#5391]
#### What are you trying to achieve?
Retrieve records with 2 parameters on one column:
```php
$I->seeNumRecords(
            0,
            'accounts',
            ['expire' => ['<=', date('Y-m-d')], 'status' => '1'] // ERROR OCCURS HERE
        );
```


#### What do you get instead?
```bash
Acceptance (appCli, noRecords) Tests (1) -------------------------------------------------------------------------------------------------------------------------------------------------------
Modules: Asserts, Cli, Filesystem, Laravel5, \Helper\Laravel5Extension
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
UpdateAccountsCest: Update accounts empty
Signature: UpdateAccountsCest:updateAccountsEmpty
Test: tests/acceptance/UpdateAccountsCest.php:updateAccountsEmpty
Scenario --
 I see num records 0,"accounts",{"expire":["<=","2019-02-17"],"status":"1"}
 ERROR 

------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------


Time: 4.22 seconds, Memory: 18.00MB

There was 1 error:

---------
1) UpdateAccountsCest: Update accounts empty
 Test  tests/acceptance/UpdateAccountsCest.php:updateAccountsEmpty
                                                                                                                                                                                                         
  [Illuminate\Database\QueryException] SQLSTATE[HY000]: General error: 25 column index out of range (SQL: select count(*) as aggregate from "accounts" where "expire" = \<= and "status" = 2019-02-17)  
                                                                                                                                                                                                         

Scenario Steps:

 1. $I->seeNumRecords(0,"accounts",{"expire":["<=","2019-02-17"],"status":"1"}) at tests/acceptance/UpdateAccountsCest.php:54

/Users/edno/Workspace/kleis/vendor/laravel/framework/src/Illuminate/Database/Connection.php:664
/Users/edno/Workspace/kleis/vendor/laravel/framework/src/Illuminate/Database/Connection.php:624
/Users/edno/Workspace/kleis/vendor/laravel/framework/src/Illuminate/Database/Connection.php:333
/Users/edno/Workspace/kleis/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:2095
/Users/edno/Workspace/kleis/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:2083
/Users/edno/Workspace/kleis/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:2569
/Users/edno/Workspace/kleis/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:2084
/Users/edno/Workspace/kleis/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:2496
/Users/edno/Workspace/kleis/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:2424
/Users/edno/Workspace/kleis/vendor/codeception/codeception/src/Codeception/Module/Laravel5.php:1108
/Users/edno/Workspace/kleis/vendor/codeception/codeception/src/Codeception/Module/Laravel5.php:1004
/Users/edno/Workspace/kleis/vendor/codeception/codeception/src/Codeception/Step.php:265
/Users/edno/Workspace/kleis/vendor/codeception/codeception/src/Codeception/Scenario.php:72
/Users/edno/Workspace/kleis/tests/_support/_generated/AcceptanceTesterActions.php:1362
/Users/edno/Workspace/kleis/tests/acceptance/UpdateAccountsCest.php:54
/Users/edno/Workspace/kleis/vendor/codeception/codeception/src/Codeception/Lib/Di.php:127
/Users/edno/Workspace/kleis/vendor/codeception/codeception/src/Codeception/Test/Cest.php:138
/Users/edno/Workspace/kleis/vendor/codeception/codeception/src/Codeception/Test/Cest.php:150
/Users/edno/Workspace/kleis/vendor/codeception/codeception/src/Codeception/Test/Cest.php:82
/Users/edno/Workspace/kleis/vendor/codeception/codeception/src/Codeception/Test/Test.php:89
/Users/edno/Workspace/kleis/vendor/phpunit/phpunit/src/Framework/TestSuite.php:746
/Users/edno/Workspace/kleis/vendor/codeception/phpunit-wrapper/src/Runner.php:118
/Users/edno/Workspace/kleis/vendor/codeception/codeception/src/Codeception/SuiteManager.php:158
/Users/edno/Workspace/kleis/vendor/codeception/codeception/src/Codeception/Codecept.php:192
/Users/edno/Workspace/kleis/vendor/codeception/codeception/src/Codeception/Codecept.php:181
/Users/edno/Workspace/kleis/vendor/codeception/codeception/src/Codeception/Command/Run.php:384
/Users/edno/Workspace/kleis/vendor/symfony/console/Command/Command.php:255
/Users/edno/Workspace/kleis/vendor/symfony/console/Application.php:901
/Users/edno/Workspace/kleis/vendor/symfony/console/Application.php:262
/Users/edno/Workspace/kleis/vendor/symfony/console/Application.php:145
/Users/edno/Workspace/kleis/vendor/codeception/codeception/src/Codeception/Application.php:108
/Users/edno/Workspace/kleis/vendor/codeception/codeception/codecept:43


ERRORS!
Tests: 1, Assertions: 0, Errors: 1.
```

### Details

* Codeception version: 2.5.2
* PHP Version: 7.3.2
* Operating System: Darwin
* Installation type: Composer
* List of installed packages (extract)
  * laravel/framework: 5.7.26
* Suite configuration:

```yml
class_name: AcceptanceTester
modules:
    enabled:
        - Asserts
        - Cli
        - Filesystem
        - Laravel5:
            environment_file: .env.codecept
            part: orm
            cleanup: false
            run_database_migrations: false
        - Helper\Laravel5Extension
```

**Comment by edno** [comment]
Fix all query builders in `src/Codeception/Module/Laravel5.php` by using `call_user_func_array` if attribute value is array type
```php
    protected function countRecords($table, $attributes = [])
    {
        $query = $this->getQueryBuilderFromTable($table);
        foreach ($attributes as $key => $value) {
            if (\is_array($value)) {
                 call_user_func_array(array($query, "where"), $value);
            } else {
                $query->where($key, $value);
            }
        }

        return $query->count();
    }
```
I will submit a PR for this fix.

**Comment by Naktibalda** [comment]
Fixed by #5484
